src/cli.py

```diff
@@ -33,16 +33,19 @@ def cli():
             return
         try:
             server, port, toSend = gs.getInput(prompt='to send: ')
-            resp = gs.transaction(server, port, toSend)
-
-            #checks if housekeeping multiple packets. if so, a list of dictionaries is returned
-            if type(resp) == list:
-                for rxData in resp:
-                    print("--------------------------------------------------------------------------")
-                    [print(key,':',value) for key, value in rxData.items()]
-            #else, only a single dictionary is returned
+            if server == 24:
+                gs.__setPIPE__()
```

Comment:
To me this looks like it sets the pipe, then doesn't do the transaction. Is that right?

Reply:
Correct, there's no CSP transaction here. This is to put the ground station transceiver and will be removed eventually since we will not need a command to put the ground radio into PIPE with the SDR

Reply:
I will add a comment indicating such